src/app/zap-templates/zcl/data-model/chip/basic-information-cluster.xml

```diff
@@ -14,159 +14,157 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    <item name="Nickel" value="15"/>
-    <item name="Chrome" value="16"/>
-    <item name="Brass" value="17"/>
-    <item name="Copper" value="18"/>
-    <item name="Silver" value="19"/>
-    <item name="Gold" value="20"/>
-  </enum>
+    <cluster code="0x0039"/>
+    <item fieldId="0" name="CaseSessionsPerFabric" type="int16u" default="3" min="3"/>
+    <item fieldId="1" name="SubscriptionsPerFabric" type="int16u" default="3" min="3"/>
+  </struct>
 
-  <struct name="ProductAppearanceStruct">
+  <struct name="ProductAppearanceStruct" apiMaturity="provisional">
     <cluster code="0x0028"/>
-    <item fieldId="0" name="Finish"       type="ProductFinishEnum" isNullable="false"/>
-    <item fieldId="1" name="PrimaryColor" type="ColorEnum"         isNullable="true" />
+    <cluster code="0x0039"/>
+    <item fieldId="0" name="Finish" type="ProductFinishEnum" min="0x00" max="0x05"/>
+    <item fieldId="1" name="PrimaryColor" type="ColorEnum" isNullable="true" min="0x00" max="0x14"/>
   </struct>
 
-  <cluster singleton="true">
-    <name>Basic Information</name>
+  <cluster>
     <domain>CHIP</domain>
+    <name>Basic Information</name>
     <code>0x0028</code>
     <define>BASIC_INFORMATION_CLUSTER</define>
-    <description>This cluster provides attributes and events for determining basic information about Nodes, which supports both
-      Commissioning and operational determination of Node characteristics, such as Vendor ID, Product ID and serial number,
-      which apply to the whole Node. Also allows setting user device information such as location.</description>
-    <globalAttribute side="either" code="0xFFFD" value="3"/>
-
-    <attribute side="server" code="0" name="DataModelRevision" define="DATA_MODEL_REVISION" type="int16u">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="1" name="VendorName" define="VENDOR_NAME" type="char_string" length="32">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="2" name="VendorID" define="VENDOR_ID" type="vendor_id">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="3" name="ProductName" define="PRODUCT_NAME" type="char_string" length="32">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="4" name="ProductID" define="PRODUCT_ID" type="int16u">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="5" name="NodeLabel" define="NODE_LABEL" type="char_string" length="32" writable="true">
-      <access op="read" privilege="view"/>
+    <description>The conformance of the UniqueID attribute was optional in cluster revisions prior to revision 4.</description>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="5"/>
+    <attribute code="0x0000" side="server" define="DATA_MODEL_REVISION" type="int16u">DataModelRevision</attribute>
```

Comment:
See https://github.com/project-chip/connectedhomeip/pull/38137#pullrequestreview-2731028892 -- there's some stuff that Alchemy does here that's suboptimal (because old Alchemy version?) and some stuff that's plain wrong.

Reply:
Thanks, let me try to re-generate with newest version.